http/task_service.go

```diff
@@ -322,7 +322,7 @@ type getTasksRequest struct {
 	filter platform.TaskFilter
 }
 
-func decodeGetTasksRequest(ctx context.Context, r *http.Request) (*getTasksRequest, error) {
+func decodeGetTasksRequest(ctx context.Context, r *http.Request, orgs platform.OrganizationService) (*getTasksRequest, error) {
```

Comment:
@lyondhill you moved some org service calls into the platform adapter in #12004. Does it make sense to move this into the platform adapter too, or should it stay here?

Reply:
TaskService `FindTasks` currently doesn't support org name. It can be moved to the platform adapter if we wanted to update the FindTasks method in the interface. 
Out of this context The FindTasks probably needs to be updated to use `FindOptions` sometime. When that happens would be a good time to move this out of the http handler.